WordPress/Resources/bg.lproj/Localizable.strings

```diff
@@ -3212,6 +3215,12 @@
 /* Text for related post cell preview */
 "in \"Upgrade\"" = "в \"Обновяване\"";
 
+/* This text is used when the user is configuring the iOS widget to suggest them to select the site to configure the widget for */
+"ios-widget.gpCwrM" = "Select Site";
+
+/* This text is required by Apple to be part of the translations but is not shown to the users. You can use the English version verbatim without translation for this entry. */
+"ios-widget.tVvJ9c" = "Select Site Intent";
```

Comment:
These should be in all the locales that didn't have them yet, and should stop ASC from sending us warning emails about missing translations for the Intent.